features/dd-sdk-android-trace/transitiveDependencies

```diff
@@ -1,15 +1,23 @@
 Dependencies List
 
 androidx.annotation:annotation:1.3.0                            :   30 Kb
+com.datadoghq:sketches-java:0.8.2                               :  125 Kb
 com.google.code.gson:gson:2.10.1                                :  276 Kb
+com.google.re2j:re2j:1.7                                        :  111 Kb
+com.squareup.moshi:moshi:1.11.0                                 :  148 Kb
+com.squareup.okhttp3:okhttp:4.11.0                              :  768 Kb
+com.squareup.okio:okio-jvm:3.2.0                                :  337 Kb
 io.opentelemetry:opentelemetry-api:1.4.0                        :   78 Kb
 io.opentelemetry:opentelemetry-context:1.4.0                    :   42 Kb
 io.opentracing:opentracing-api:0.32.0                           :   18 Kb
 io.opentracing:opentracing-noop:0.32.0                          :   10 Kb
 io.opentracing:opentracing-util:0.32.0                          :   10 Kb
+org.jctools:jctools-core:3.3.0                                  :  328 Kb
 org.jetbrains.kotlin:kotlin-stdlib-common:1.8.10                :  212 Kb
+org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.8.10                  :  963 b 
+org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.8.10                  :  969 b 
 org.jetbrains.kotlin:kotlin-stdlib:1.8.10                       : 1598 Kb
```

Comment:
Ah :
```
com.squareup.okhttp3:okhttp:4.11.0
|    +--- com.squareup.okio:okio:3.2.0 (*)
|    +--- org.jetbrains.kotlin:kotlin-stdlib:1.6.20 -> 1.8.10 (*)
|    \--- org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.6.20 -> 1.8.10 
```
From `okhttp`

Reply:
ok, they come because of the version bump done by Gradle. In theory we can remove those from dependencies explicitly, because all classes from `-jdk7` and `-jdk8` are in the main `stdlib`. But here they (these JDK7 and JDK8) they are probably empty by looking on the size, so no risk of getting `DuplicateClassException`.

Reply:
yeah...I think I am going to try to do more cleaning at the end. At least `moshi` I want to remove it from there. It is going to be a big mess that's the reason why I want to do it at the end.